app.js

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+'use strict';
+
+const http      = require('http');
+const socketio  = require('socket.io');
+const Kasocki   = require('kasocki');
+const P         = require('bluebird');
```

Comment:
Minor: Most style guides discourage this alignment (as it's hard to maintain), but that's a personal preference we don't enforce.